.eslintrc.js

```diff
@@ -197,6 +197,11 @@ module.exports = {
                 from: ['src/core/utils/**/*'],
                 errorMessage: `Plugins may only import from src/core/server and src/core/public.`,
               },
+              {
+                target: ['(src|x-pack)/plugins/*/server/**/*', '!x-pack/plugins/apm/**/*'],
```

Comment:
APM is a less than straightforward update and I would rather not make assumptions for their desired organization. I will follow-up with an issue for the team to resolve.

Reply:
I suppose this is caused by the server side types consumed by the client? Types are inferred from server-side routes and can therefore not be moved to common.

I expect we can resolve this when we update to typescript 3.8 and get [type only imports](https://github.com/microsoft/TypeScript/pull/35200).

Reply:
cc @dgieselaar